diagnostic.md

```diff
@@ -7,40 +7,44 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:noobiwankenoobi/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+You make sure you move into the correct directory and that it says (master) next to it.
```

Comment:
You definitely need to do this. We were looking for the answer 'create a new branch', but you need to do this step first